openff/interchange/components/mdconfig.py

```diff
@@ -18,13 +20,15 @@
 if TYPE_CHECKING:
     from openff.interchange import Interchange
 
-MDP_HEADER = """
-nsteps                   = 0
-nstenergy                = 1000
-continuation             = yes
-cutoff-scheme            = verlet
+MDP_HEADER = """\
+; Single point energy
+nsteps                   = 0      ; Perform no MD steps
+nstenergy                = 1000   ; Write energies to .EDR file
+continuation             = yes    ; Do not apply constraints at start
+cutoff-scheme            = verlet ; Use a Verlet pair list for cutoffs
```

Comment:
Not worth doing here, but this file screams out to be stored in memory with a little class that inherits from `NamedTuple` or something

Reply:
Mmm I was thinking about something like that